### PR TITLE
Document antrea

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: src/images/icons/antrea.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (areed) [inline src/images/icons/antrea.svg:1]: This includes the word 'Antrea' in it, but we just want the icon.

File: src/components/Kurlsh.js [hunk 2]
```diff
@@ -926,34 +936,39 @@ class Kurlsh extends React.Component {
...
                   <div className="flex flex1">
                     <div className="flex flex1 alignItems--center">
                       <input
                         type="checkbox"
                         className="u-marginRight--normal"
-                        checked={selectedVersions.calico.version !== "None"}
+                        checked={selectedVersions.antrea.version !== "None"}
                       />
-                      <span className="icon u-calico u-marginBottom--small" />
+                      <span className="icon u-velero u-marginBottom--small" />
```
Comment (areed) [inline src/components/Kurlsh.js:947]: Velero is the closest icon to Antrea. Need to figure out how to add the antrea icon to the spritesheet, then change this to `u-antrea`.

File: src/images/icons/antrea.svg (hunk 1 above)
Comment (GraysonNull) [reply src/images/icons/antrea.svg:1]: Plus one here

Comment (areed) [reply src/images/icons/antrea.svg:1]: I'll remove this file since it's on the spritesheet

**Review by GraysonNull — APPROVED** [review]

**Review by jgruica — APPROVED** [review]

**Pull request merged**